charts/kubernetes/templates/applications.yaml

```diff
@@ -162,6 +162,9 @@ spec:
   tags:
   - infrastructure
   versions:
+  - parameters:
```

Comment:
Close, `parameters` is a key under a version, see the use of it else where, and try keep the ordering of the properties vaguely consistent, my OCD eyes will pick out odd formatting!

Reply:
haha no worries. to be honest I was pushing to do some testing but then colima cocked up due to a bug so might be worth ignoring this PR for now until testing is done ;-)

Reply:
Also face palming at my error there. It's quite, quite clear it's part of the version itsel and not another list within the object 🤦